vendor/build-logger/src/ldlogger-tool-gcc.c

```diff
@@ -227,6 +231,23 @@ static void getDefaultArguments(const char* prog_, LoggerVector* args_)
   pclose(cmdOut);
 }
 
+char* findFullPath(const char* executable, char* fullpath) {
```

Comment:
Isn't there already a posix function to do this work? (E.g.: `realpath`?)

Reply:
I did not find any. `realpath` only resolves paths that contain `.` or `..` or are symbolic links, but it does not search in the `$PATH` env variable.